typescript/infra/scripts/generate-renzo-pzeth-warp-route-config.ts

```diff
@@ -0,0 +1,23 @@
+import { writeFileSync } from 'fs';
```

Comment:
nit: I'm not sure we need this script, I think it was used in the ezETH case to manually hand craft the config by Nam. I assume we are using the CLI now to generate the config. I would just leave it in for now and we can reassess later

Reply:
yea, we can use the CLI, but wanted to match what was done with ezEth.